source/fundamentals/connection/mongoclientsettings.txt

```diff
@@ -485,18 +509,26 @@ to modify the driver's behavior:
      - Uses the socket settings specified in a ``SocketSettings`` object.
 
    * - ``connectTimeout()``
-     - Sets the maximum time to connect to an available socket before throwing
-       a timeout exception.
+     - | Sets the maximum time to connect to an available socket before throwing
+       | a timeout exception.
+       |
+       | **Default**: ``10 seconds``
 
    * - ``readTimeout()``
-     - Sets the maximum time to read to an available socket before throwing a
-       timeout exception.
+     - | Sets the maximum time to read from an available socket before throwing a
+       | timeout exception.
+       |
+       | **Default**: ``0 seconds``
```

Comment:
This might be a little misleading, without adding something like 0 indicating no read timeout at all.